cql3/statements/select_statement.cc

```diff
@@ -1318,21 +1328,34 @@ indexed_table_select_statement::find_index_clustering_rows(service::storage_prox
 
 namespace raw {
 
+static void validate_attrs(const cql3::attributes::raw& attrs) {
+    if (attrs.timestamp) {
+        throw exceptions::invalid_request_exception("Specifying TIMESTAMP is not legal for SELECT statement");
+    }
```

Comment:
I can imagine pushing MVCC into sstables and allowing queries in a different time...

Reply:
Yup, one day. Meanwhile, the grammar will be ready for it.